shanoir-ng-studies/src/main/java/org/shanoir/ng/center/controler/CenterApiController.java

```diff
@@ -84,12 +85,11 @@
 	@Override
 	public ResponseEntity<CenterDTO> findCenterById(
 			@ApiParam(value = "id of the center", required = true) @PathVariable("centerId") final Long centerId) {
-		
-		final Center center = centerService.findById(centerId);
-		if (center == null) {
+		final Optional<Center> center = centerService.findById(centerId);
+		if (center.isEmpty()) {
 			return new ResponseEntity<>(HttpStatus.NOT_FOUND);
 		}
-		return new ResponseEntity<>(centerMapper.centerToCenterDTO(center), HttpStatus.OK);
+		return new ResponseEntity<>(centerMapper.centerToCenterDTO(center.orElseThrow()), HttpStatus.OK);
```

Comment:
I think, in general it is recommended to prefer always .orElseThrow() instead of .get(), as get might be empty. I agree, the above example is already checked, but I would prefer to keep my overall pattern in general: to be less error prown with upcoming code changes, and the result is the same.